src/server/node_func.c

```diff
@@ -2416,15 +2430,28 @@ static int	nsockets;
  *
  * @param[in]	data	-	not used here
  * @param[in]	elem	-	compares with value "Node"
- * @param[in]	attr	-	not used here
+ * @param[in]	attr	-	attribute
  *
  * @return	void
  */
 static void XMLCALL
-CrayXMLstart(void *data, const char *elem, const char **attr)
-{
-	if (!strcmp(elem, "Node"))
-		nsockets += 2;	/* two sockets per node */
+CrayXMLstart(void *data, const char *elem, const char **attr) {
+	char *pc;
+	if (!strcmp(elem, "BasilResponse") && attr[0] && !strcmp(attr[0],
```

Comment:
All of these strings (like "BasilResponse") should be defined as macros in basil.h. Please do not use the strings directly.